lib/app/views/onc_program.erb

```diff
@@ -553,9 +553,9 @@
               label: 'Limit validation to a maximum resource count',
               description: %(
                 While testing we need to validate resources in downloaded NDJSON file.
-                Please enter the number of lines/resources that would be validated for each NDJSON file. 
+                Please enter a number to limit the number of resoruces validated per resource type.                 
```

Comment:
Typo on `resources` and I don't think you need the `To skip resource validation...` line, I feel like limiting to 0 is good enough.

Reply:
Good catch. Fixed wording.